src/driver/collection.ts

```diff
@@ -101,23 +108,54 @@ export class Collection extends MongooseCollection {
   }
 
   // No-ops
-  aggregate() {
-    throw new Error('aggregate() Not Implemented');
+  bulkWrite(ops: any[], options?: any) {
+    throw new Error('bulkWrite() Not Implemented');
```

Comment:
Should also be an OperationNotSupportedError here no?

Reply:
Fixed this.